pkg/auth/auth.go

```diff
@@ -34,8 +35,22 @@ func CheckAuthFile(authfile string) error {
 }
 
 // Login login to the server with creds from Stdin or CLI
```

Comment:
```suggestion
// Login implements a “log in” command with the provided opts and args, reading the password from opts.Stdin or the options in opts.
```

Applies similarly to the logout code.